aurelia-slickgrid/src/aurelia-slickgrid/editors/integerEditor.ts

```diff
@@ -34,7 +36,7 @@ export class IntegerEditor implements Editor {
   }
 
   loadValue(item: any) {
-    this.defaultValue = item[this.args.column.field];
+    this.defaultValue = parseInt(item[this.args.column.field], 10);
```

Comment:
without this `parseInt`, the `isValueChanged` function was returning `True` when clicking on Inline Editor without changing the value, because the string was not equal to a number and so this was also triggering a `onCellChanged` which was unwanted. It was mostly obvious with a "0" in the input and I didn't want that behavior. You can see this by modifying the Example 3 and connecting the `onCellChanged`

I'm not sure why but I only had to do this code change in the Integer Editor, not the Float Editor.

Reply:
@jmzagorski 
are you ok with the change? I'd like to get this PR merged by the end of week. I'm planning on a new release during the weekend. Thanks

Reply:
Yes, looks good from what I can see.

Reply:
yes looks good from what i can see